src/unix/darwin-stub.h

```diff
@@ -110,4 +111,9 @@ static const int kFSEventStreamEventFlagRootChanged = 32;
 static const int kFSEventStreamEventFlagUnmount = 128;
 static const int kFSEventStreamEventFlagUserDropped = 2;
 
+// Copied from https://opensource.apple.com/source/xnu/xnu-6153.101.6/libsyscall/wrappers/spawn/spawn_private.h.auto.html
```

Comment:
```suggestion
/* Copied from https://opensource.apple.com/source/xnu/xnu-6153.101.6/libsyscall/wrappers/spawn/spawn_private.h.auto.html */
```
I.e., no C99/C++ style comments. Also applies to other files.

Also, as a general rule, wrap lines > 80 columns.